subsys/bluetooth/services/cgms/cgms_internal.h

```diff
@@ -0,0 +1,144 @@
+ */
+#ifndef BT_CGMS_INTERNAL_H_
+#define BT_CGMS_INTERNAL_H_
+
+#include <zephyr/types.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <bluetooth/services/cgms.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* Minimum communication interval in minute */
+#define BT_CGMS_COMM_INTERVAL_MIN               1
+
+/* Continuous Glucose Monitoring feature */
+#define BT_SERVICE_CGMS_FEAT_CALIBRATION_SUPPORTED                           (0x01 << 0)
+#define BT_SERVICE_CGMS_FEAT_PATIENT_HIGH_LOW_ALERTS_SUPPORTED               (0x01 << 1)
+#define BT_SERVICE_CGMS_FEAT_HYPO_ALERTS_SUPPORTED                           (0x01 << 2)
+#define BT_SERVICE_CGMS_FEAT_HYPER_ALERTS_SUPPORTED                          (0x01 << 3)
+#define BT_SERVICE_CGMS_FEAT_RATE_OF_INCREASE_DECREASE_ALERTS_SUPPORTED      (0x01 << 4)
+#define BT_SERVICE_CGMS_FEAT_DEVICE_SPECIFIC_ALERT_SUPPORTED                 (0x01 << 5)
+#define BT_SERVICE_CGMS_FEAT_SENSOR_MALFUNCTION_DETECTION_SUPPORTED          (0x01 << 6)
+#define BT_SERVICE_CGMS_FEAT_SENSOR_TEMPERATURE_HIGH_LOW_DETECTION_SUPPORTED (0x01 << 7)
+#define BT_SERVICE_CGMS_FEAT_SENSOR_RESULT_HIGH_LOW_DETECTION_SUPPORTED      (0x01 << 8)
+#define BT_SERVICE_CGMS_FEAT_LOW_BATTERY_DETECTION_SUPPORTED                 (0x01 << 9)
+#define BT_SERVICE_CGMS_FEAT_SENSOR_TYPE_ERROR_DETECTION_SUPPORTED           (0x01 << 10)
+#define BT_SERVICE_CGMS_FEAT_GENERAL_DEVICE_FAULT_SUPPORTED                  (0x01 << 11)
+#define BT_SERVICE_CGMS_FEAT_E2E_CRC_SUPPORTED                               (0x01 << 12)
+#define BT_SERVICE_CGMS_FEAT_MULTIPLE_BOND_SUPPORTED                         (0x01 << 13)
+#define BT_SERVICE_CGMS_FEAT_MULTIPLE_SESSIONS_SUPPORTED                     (0x01 << 14)
+#define BT_SERVICE_CGMS_FEAT_CGM_TREND_INFORMATION_SUPPORTED                 (0x01 << 15)
+#define BT_SERVICE_CGMS_FEAT_CGM_QUALITY_SUPPORTED                           (0x01 << 16)
+
+/* CGM Measurement Sensor Status Annunciation */
+#define BT_SERVICE_CGMS_STATUS_SESSION_STOPPED                  (0x01 << 0)
+#define BT_SERVICE_CGMS_STATUS_DEVICE_BATTERY_LOW               (0x01 << 1)
+#define BT_SERVICE_CGMS_STATUS_SENSOR_TYPE_INCORRECT_FOR_DEVICE (0x01 << 2)
+#define BT_SERVICE_CGMS_STATUS_SENSOR_MALFUNCTION               (0x01 << 3)
+#define BT_SERVICE_CGMS_STATUS_DEVICE_SPECIFIC_ALERT            (0x01 << 4)
+#define BT_SERVICE_CGMS_STATUS_GENERAL_DEVICE_FAULT             (0x01 << 5)
+
+/* CGM Measurement flags */
+#define BT_SERVICE_CGMS_FLAG_TREND_INFO_PRESENT          0x01
+#define BT_SERVICE_CGMS_FLAGS_QUALITY_PRESENT            0x02
+#define BT_SERVICE_CGMS_STATUS_FLAGS_WARNING_OCT_PRESENT 0x20
+#define BT_SERVICE_CGMS_STATUS_FLAGS_CALTEMP_OCT_PRESENT 0x40
+#define BT_SERVICE_CGMS_STATUS_FLAGS_STATUS_OCT_PRESENT  0x80
+
+struct bt_cgm_sensor_annunc_t {
+	/* Warning annunciation. */
+	uint8_t		warning;
+	/* Calibration and Temperature annunciation. */
+	uint8_t		calib_temp;
+	/* Status annunciation. */
+	uint8_t		status;
```

Comment:
```suggestion
	uint8_t warning;
	/* Calibration and Temperature annunciation. */
	uint8_t calib_temp;
	/* Status annunciation. */
	uint8_t status;
```
The same in other places.